content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+{{< feature-state for_k8s_version="v1.19" state="alpha" >}}
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
+Tracking storage capacity is supported for [{{< glossary_tooltip
+text="Container Storage Interface" term_id="csi" >}} (CSI) drivers](/docs/concepts/storage/volumes/#csi) and
```

Comment:
Should we link to the page where we have a list of csi drivers and their feature support? https://kubernetes-csi.github.io/docs/drivers.html

Reply:
Yes, that's better here. We have another cross-reference to /docs/concepts/storage/volumes/#csi further down. Changed.